### PR TITLE
Onion2

### PR DESCRIPTION
**Pull request by IsaacLuo**
Applied Anais's styles for existing components.
fixed some bugs for selecting.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -2,6 +2,7 @@
 <html>
   <head>
     <title>Genome Designer</title>
+    <link href='http://fonts.googleapis.com/css?family=Cousine' rel='stylesheet' type='text/css'>
```
Comment (maxbates) [inline src/index.html]: Please add this to your extension, not to the app. you can programmatically add this to meta field?

File: webpack.config.dev.js [hunk 2]
```diff
@@ -32,7 +32,11 @@ module.exports = Object.assign({}, webpackBase, {
         exclude: /node_modules/
       },
       {
-        test  : /\.css$/,
+        test: /\.css$/,
+        loader: 'style-loader!css-loader!postcss-loader'
+      },
+      {
+        test:/\.scss$/,
```
Comment (maxbates) [inline webpack.config.dev.js]: this is incorrect - should use a scss loader. what SCSS is there?

File: src/onionExtension.js [hunk 3]
```diff
@@ -4,6 +4,17 @@ import { render as reactRender } from 'react-dom';
 import {SequenceEditor} from './extensions/onion2/SequenceEditor';
 import {onionFile} from './extensions/onion2/OnionFile';
 import {PlasmidViewer} from './extensions/onion2/PlasmidViewer';
+import {InfoBar} from './extensions/onion2/InfoBar';
+import {Emzyme, loadEnzymeList} from './extensions/onion2/Bio/Enzyme';
+import {MenuBar} from './extensions/onion2/MenuBar';
+
+var $ = require('jquery');
```
Comment (maxbates) [inline src/onionExtension.js]: Don't add dependencies to the main app - you can load them in yourself. but they shouldn't be in package.json... is there any reason they need to be?

File: webpack.config.dev.js [hunk 4]
```diff
@@ -32,8 +32,16 @@ module.exports = Object.assign({}, webpackBase, {
         exclude: /node_modules/
       },
       {
-        test  : /\.css$/,
+        test: /\.css$/,
         loader: 'style-loader!css-loader!postcss-loader'
+      },
+      {
+        test:/\.scss$/,
```
Comment (maxbates) [inline webpack.config.dev.js]: why?

File: src/styles/GoogleFonts.css [hunk 5]
```diff
@@ -0,0 +1,64 @@
+/* cyrillic-ext */
```
Comment (maxbates) [inline src/styles/GoogleFonts.css]: This also should be loaded in by your plugin directly

File: src/onionExtension.js (hunk 3 above)
Comment (IsaacLuo) [inline src/onionExtension.js]: I moved code from onionExtension.js to OnionForGenomeDesigner.js, a file in the extensions folder.

File: src/styles/GoogleFonts.css (hunk 5 above)
Comment (IsaacLuo) [inline src/styles/GoogleFonts.css]: moved

**Comment by dchandran** [comment]
@maxbates @IsaacLuo Did someone ok this pull request? Just checking.

**Comment by dchandran** [comment]
@maxbates @IsaacLuo Never mind. I think it looks ok generally.

**Pull request merged**